.github/workflows/ci-backend.yml

```diff
@@ -289,13 +284,14 @@ jobs:
                   fi
 
             - name: Fail CI if some snapshots have been updated but not committed
-              if: steps.changed-files.outputs.files_found == 'true' && steps.add-and-commit.outcome == 'success'
+              if: ${{ matrix.clickhouse-server-image == 'clickhouse/clickhouse-server:22.3' && !matrix.person-on-events && steps.changed-files.outputs.files_found == 'true' && steps.add-and-commit.outcome == 'success' }}
```

Comment:
hmm, I think you don't need this because `steps.add-and-commit.outcome` handles the `if` condition? Was this not happening?

Reply:
This way this conditional needs to be in one place only

Reply:
I was worried this would error out as technically the step would be "undefined" then but turns out this is handled fine, so cool, thanks!

Reply:
Yep, in that case the outcome is `skipped`.